DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SendBinding.cs

```diff
@@ -82,7 +78,9 @@ private async Task HandleSend(string modelCardId)
       throw new InvalidOperationException("No publish model card was found.");
     }
 
-    string versionId = await _sendOperation
+    using IUnitOfWork<SendOperation> sendOperation = _unitOfWorkFactory.Resolve<SendOperation>();
```

Comment:
Unless we have a powerful reason not to, we should declare the unit of work as the first thing in the `HandleSend` method.

This will make it very clear that anything inside this is considered it's own bubble. Having it a couple lines down makes it less obvious.

Reply:
I'm not understanding the benefits of this. "This will make it very clear that anything inside this (referring to the Method, I think?) is considered it's own bubble. Having it a couple lines down makes it less obvious". 

The unit of work _IS_ its own bubble. By bubble I am referring to a separate IOC container scope. This is somewhat enforced by the IDisposable implementation. The stuff above is validation logic that isn't part of the same scope. The CancellationTokenSource object's lifetime is completely different to the lifetime of the SendOperation, and dragging it down into the disposable block makes it seem like it has the same lifetime which isn't true. It's a pretty trivial difference, but I don't see any benefit to creating this object and its scope until we need it.